php_gearman_client.c

```diff
@@ -113,7 +113,7 @@ void gearman_client_free_obj(zend_object *object) {
         zend_object_std_dtor(&intern->std);
 }
 
-/* {{{ proto int gearman_client_return_code()
+/* {{{ proto ?int gearman_client_return_code()
```

Comment:
As of PHP 8.0 ZPP always throws an error, and ZPP failures are not usually described (as it is Undefined Behaviour prior to PHP 8.0).

This is relevant for the documentation, as we do not document ZPP failures.

Reply:
Ok, I was not sure about that. Thanks for clarifying. I will revert all affected changes.